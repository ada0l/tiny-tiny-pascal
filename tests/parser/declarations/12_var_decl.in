var a: Integer = 10; b: Real = 1000.0;

begin end.
