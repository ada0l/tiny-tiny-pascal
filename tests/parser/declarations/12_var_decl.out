main
   program name:
      without name
   decls:
      vars:
         var
            a 
            type:
               Integer 
            10
         var
            b 
            type:
               Real 
            1000.0
   block:
      compounds statements